[devil] Build error on x64-freebsd

Hello, devil needs an update with removal of register keyboard cause while i use ./vcpkg install devil i got a problem with 



/home/vcpkg/buildtrees/devil/src/v1.8.0-13a02e996b.clean/DevIL/src-IL/src/il_manip.cpp:40:2: error: ISO C++17 does not allow 'register' storage class specifier [-Wregister]
        register int s =  (i >> 16) & 0x00008000;
        ^~~~~~~~~
/home/vcpkg/buildtrees/devil/src/v1.8.0-13a02e996b.clean/DevIL/src-IL/src/il_manip.cpp:41:2: error: ISO C++17 does not allow 'register' storage class specifier [-Wregister]
        register int e = ((i >> 23) & 0x000000ff) - (127 - 15);
        ^~~~~~~~~
/home/vcpkg/buildtrees/devil/src/v1.8.0-13a02e996b.clean/DevIL/src-IL/src/il_manip.cpp:42:2: error: ISO C++17 does not allow 'register' storage class specifier [-Wregister]
        register int m =   i        & 0x007fffff;
        ^~~~~~~~~

--- a/ports/devil/0007-remove_register_keyword_cpp17.patch
+++ b/ports/devil/0007-remove_register_keyword_cpp17.patch
@@ -1,0 +1,30 @@
+diff --git a/DevIL/src-IL/src/il_manip.cpp b/DevIL/src-IL/src/il_manip.cpp
+index 79acc99..229a692 100644
+--- a/DevIL/src-IL/src/il_manip.cpp
++++ b/DevIL/src-IL/src/il_manip.cpp
+@@ -37,9 +37,9 @@ ILushort ILAPIENTRY ilFloatToHalf(ILuint i) {
+ 	// of float and half (127 versus 15).
+ 	//
+ 
+-	register int s =  (i >> 16) & 0x00008000;
+-	register int e = ((i >> 23) & 0x000000ff) - (127 - 15);
+-	register int m =   i        & 0x007fffff;
++	int s =  (i >> 16) & 0x00008000;
++	int e = ((i >> 23) & 0x000000ff) - (127 - 15);
++	int m =   i        & 0x007fffff;
+ 
+ 	//
+ 	// Now reassemble s, e and m into a half:
+diff --git a/DevIL/src-ILU/src/ilu_scaling.cpp b/DevIL/src-ILU/src/ilu_scaling.cpp
+index c2893a3..ef35c13 100644
+--- a/DevIL/src-ILU/src/ilu_scaling.cpp
++++ b/DevIL/src-ILU/src/ilu_scaling.cpp
+@@ -406,7 +406,7 @@ main(argc, argv)
+ int argc;
+ char *argv[];
+ {
+-	register int c;
++	int c;
+ 	int optind;
+ 	char *optarg;
+ 	int xsize = 0, ysize = 0;

--- a/ports/devil/portfile.cmake
+++ b/ports/devil/portfile.cmake
@@ -13,6 +13,7 @@ vcpkg_from_github(
         0005-fix-pkgconfig.patch
         0006-fix-ilut-header.patch
         jasper-4.patch
+        0007-remove_register_keyword_cpp17.patch
 )
 
 file(REMOVE "${SOURCE_PATH}/DevIL/src-IL/cmake/FindOpenEXR.cmake")

--- a/ports/devil/vcpkg.json
+++ b/ports/devil/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "devil",
   "version": "1.8.0",
-  "port-version": 11,
+  "port-version": 12,
   "description": "A full featured cross-platform image library",
   "homepage": "https://github.com/DentonW/DevIL",
   "supports": "!uwp",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2194,7 +2194,7 @@
     },
     "devil": {
       "baseline": "1.8.0",
-      "port-version": 11
+      "port-version": 12
     },
     "dimcli": {
       "baseline": "7.1.0",

--- a/versions/d-/devil.json
+++ b/versions/d-/devil.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "1e22a47b75583772a690e6e361fcbb3739838282",
+      "version": "1.8.0",
+      "port-version": 12
+    },
+    {
       "git-tree": "03010167ad1849c5f1a8e5bc044115b3c84478a3",
       "version": "1.8.0",
       "port-version": 11

